src/App.tsx

```diff
@@ -1,9 +1,42 @@
 import './App.scss';
+import {
+  Routes, Route, NavLink, Navigate,
+} from 'react-router-dom';
+import PeoplePage from './components/PeoplePage/PeoplePage';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App:React.FC = () => {
+  return (
+    <div className="App subtitle is-3">
+      <nav className="nav">
+        <NavLink
+          className={({ isActive }) => (isActive
+            ? 'button is-link is-active' : 'button is-warning')}
+          to="/"
+        >
+          Home page
+        </NavLink>
+        <NavLink
+          className={({ isActive }) => (isActive
+            ? 'button is-link is-active' : 'button is-warning')}
+          to="/people"
+        >
+          People page
+        </NavLink>
+      </nav>
+      <Routes>
+        <Route path="/" element={<h1>Home page</h1>} />
+        <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
use `replace` prop for your `navigate` to avoid infinite cycle when you press back button